### ISSUE TITLE
Wrong output for eql assertion error on arrays

### ISSUE THREAD

**Issue by targos** [shouldjs/should.js#117]
Hi.

For the following assertion: 
```js
const toTest = [
    0, 1, 0, 4,
    0, 0, 0, 3,
    0, 4, 5, 0,
    0, 0, 0, 6,
];

const expected = [
    0, 1, 0, 3,
    0, 0, 0, 3,
    0, 4, 5, 0,
    0, 0, 0, 6,
];

toTest.should.eql(expected);
```

I get the following error output:
```
       expected Array [ 0, 1, 5, 0, 0, 0, 0, 6, 0, 4, 0, 0, 0, 3, 0, 4 ] to equal Array [ 0, 1, 5, 0, 0, 0, 0, 6, 0, 3, 0, 0, 0, 3, 0, 4 ] (at '3', A has 4 and B has 3)
       
       + expected - actual
       
          0,
          0,
          6,
          0,
       -  4,
       +  3,
          0,
          0,
          0,
          3,

```

You can see that while the error is correct (including the element's index), the printed arrays do not look like the original ones. This makes it difficult to debug the tested code.

## Node.js version?
6.3.0
## Should.js version?
9.0.2

**Comment by btd** [comment]
Hi, I am very grateful for this bug report. It was problem with formating module (should-format). I published 10.0.0 (since it could be potentially breaking)

**Comment by targos** [comment]
Thanks a lot for the fix!
